hbase-common/src/main/java/org/apache/omid/transaction/CellUtils.java

```diff
@@ -227,24 +252,26 @@ private static boolean endsWith(byte[] value, int offset, int length, byte[] suf
         return result == 0;
     }
 
+    private static boolean startsWith(byte[] value, int offset, int length, byte[] prefix) {
```

Comment:
Can you just use Bytes.startsWith() instead?

Reply:
No because Bytes.startWith doesnt work with offset